chartlets.py/my_extension/my_panel_1.py

```diff
@@ -79,5 +83,45 @@ def make_figure(ctx: Context, selected_dataset: int = 0) -> alt.Chart:
         .properties(width=290, height=300, title="Vega charts")
         .add_params(corner_var, click_param)
     )
-
     return chart
+
+
+@panel.callback(
+    Input("plot", property="points"), State("plot", "chart"), Output("plot", "chart")
+)
+def get_click_event_points(ctx: Context, points, plot) -> alt.Chart:
```

Comment:
This is not the intended way to implement a callback.

- The input should not be the hole Vega chart object. They can be very large (data!) and need to be transported from server to UI on evary callback!
- The output in this case should also not b a chart, but a property within the chart, here `Output("plot", "chart.encoding.color")`
- Name your arguments according to the properties, not `plot`, but `chart`
- Add type annotations if you know the types

Reply:
Thanks for the feedback. Have done the changes.